packages/types/types/index.d.ts

```diff
@@ -193,18 +194,73 @@ export interface ThemeProviderProps {
   customBreakpoints?: string[]
 }
 
+export interface DividerProps
+  extends ColorProps,
+    SpaceProps,
+    WidthProps,
+    BorderColorProps {}
+
+export interface AvatarProps extends ClassNameProps, TextColorProps {
+  /** Heading displayed with avatar */
+  title?: string,
+  /** Subheading displayed beneath heading */
+  subtitle?: string,
+  src?: string,
+  /** Alternative initials to display when src is not provided */
+  initials?: string,
+  /** Size in px to use for height, width, and background-size */
+  size?: number
+}
+
+export interface BannerProps extends BoxProps {
+  header?: string | React.ComponentElement<IconProps, any>,
+  icon?: React.ComponentElement<IconProps, any>,
```

Comment:
See comment below about `typeof Icon`, but for line 216 we can change it to:

```
  header?: string | ReactNode,
```

That's because `header` accepts any node, but we want to restrict `icon` to just `Icon`s.

Reply:
Makes sense, updated 👍